consensus/polybft/fsm.go

```diff
@@ -406,14 +406,13 @@ func (f *fsm) VerifyStateTransactions(transactions []*types.Transaction) error {
 }
 
 // Insert inserts the sealed proposal
-func (f *fsm) Insert(proposal []byte, committedSeals []*messages.CommittedSeal) (*types.Block, error) {
-	newBlock := f.target.Block
-	receipts := f.target.Receipts
+func (f *fsm) Insert(proposal []byte, committedSeals []*messages.CommittedSeal) (*types.FullBlock, error) {
+	newBlock := &types.FullBlock{Block: f.target.Block, Receipts: f.target.Receipts}
```

Comment:
```suggestion
	newBlock := f.target
```

Reply:
Are any of the changes in this function necessary or just code style?

Reply:
Since `OnBlockInserted` is called both from `consensus` and `syncer`, I changed the `fsm.Insert` function to return a `FullBlock`, since we will need receipts from both sources in the `OnBlockInserted` (to get the exit events).